apps/backoffice-v2/src/common/env/schema.ts

```diff
@@ -17,4 +17,5 @@ export const EnvSchema = z.object({
     .or(z.literal(false))
     .catch(undefined),
   VITE_IMAGE_LOGO_URL: z.string().optional(),
+  WEBSOCKET_URL: z.string().url().default('ws://localhost:3500'),
```

Comment:
I added this variable in the 2 `.env` files but it still throw `❌ Invalid environment variables` so until I will figure it out

Reply:
Hey @TzlilSwimmer123 , there's a chance `zod` doesn't consider `ws://` a valid URL. We can use a regex match here or a custom validation doing a string comparison i.e. `url.startsWith('ws://')` and check if anything past `ws://` is a valid URL. Great job on this PR!

Reply:
@Omri-Levy thanks:)

Reply:
@Omri-Levy I checked it and have 2 comments:
1. if I put `.default('sdfsdf')` - it fails. It means zod consider it as a valid url
2. I printed the problematic env and this is it:
```
{
[1]     "VITE_API_URL": "http://localhost:3000/api/v1/internal",
[1]     "VITE_API_KEY": "demo",
[1]     "VITE_AUTH_ENABLED": "true",
[1]     "VITE_MOCK_SERVER": "false",
[1]     "VITE_POLLING_INTERVAL": "3",
[1]     "VITE_ASSIGNMENT_POLLING_INTERVAL": "5",
[1]     "VITE_EXAMPLE_TYPE": "kyb",
[1]     "BASE_URL": "/",
[1]     "MODE": "development",
[1]     "DEV": true,
[1]     "PROD": false,
[1]     "SSR": false
[1]   }
```
I didn't find where it comes from :/

Reply:
anyway, I removed it and not it's ok 🦤